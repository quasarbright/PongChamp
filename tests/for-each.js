let x = [1, 2, 3, 4];
for (let y of x) {
    println(y);
}
