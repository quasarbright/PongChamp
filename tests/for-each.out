CNumber 1
CNumber 2
CNumber 3
CNumber 4
